### ISSUE TITLE
TS2688: Cannot find type definition file for 'eslint-plugin-figma-plugins'

### ISSUE THREAD

**Issue by andrii-bodnar** [figma/eslint-plugin-figma-plugins#24]
After installing the plugin I get the following error when trying to build my plugin:

```console
ERROR in tsconfig.json
[tsl] ERROR
      TS2688: Cannot find type definition file for 'eslint-plugin-figma-plugins'.
  The file is in the program because:
    Entry point for implicit type library 'eslint-plugin-figma-plugins'
ts-loader-default_e3b0c44298fc1c14

webpack 5.76.0 compiled with 1 error in 6488 ms
```

Tsconfig:

```json
{
  "compilerOptions": {
    "target": "es6",
    "jsx": "react",
    "moduleResolution": "node",
    "allowSyntheticDefaultImports": true,
    "resolveJsonModule": true,
    "typeRoots": ["./node_modules/@types", "./node_modules/@figma"]
  }
}
```

**Comment by lauthieb** [comment]
+1, I'm blocked to build my plugin also :(

**Comment by jefflee-figma** [comment]
@lauthieb Are you also using webpack for your build?

**Comment by jefflee-figma** [comment]
I wonder if this is a TypeScript version issue. One thing that our `package.json` doesn't do is explicitly name the type definition file. If anyone is willing to experiment, can you open your project's `node_modules/@figma/eslint-plugin-figma-plugins/package.json` and make the following modification:

```
{
  ...
  "main": "dist/index.js",
  "types": "dist/index.d.ts", // <-- ADD THIS LINE (but don't add the comment)
  ...
}
```

In other words, update your local copy of the ESLint plugin with an explicit `"types"` key in its `package.json`. Does that fix it for you? If so, we can make the change to the package.

Also, can you folks let me know which version of TypeScript your project is using?

**Comment by andrii-bodnar** [comment]
@jefflee-figma thank you, the suggested modification works. Raised a PR with the fix - #25.

My TypeScript version is 4.

**Comment by lauthieb** [comment]
Hello! 

@jefflee-figma, my TypeScript version is 4.9. I use Vite for bundling.

I confirm this fix works! I've just approved the PR from @andrii-bodnar, thanks a lot! 🙏 

@jefflee-figma can you please merge it & release a new version soon? Thank you very much!

**Comment by jefflee-figma** [comment]
Thanks for the help everyone! We didn't test this plugin on TS 4 nor webpack, so I'm betting some mix of the two is what explains why we didn't catch this earlier.

**Comment by jefflee-figma** [comment]
Published changes in v0.15.0.

**Comment by lauthieb** [comment]
Thanks @jefflee-figma, I confirm, now it works since v0.15.0! 🎉
